modules/contact/doc/content/source/constraints/MechanicalContactConstraint.md

```diff
@@ -1 +1,57 @@
+\begin{equation*}
+\boldsymbol{t} = t_N\boldsymbol{n} + t_{T}\boldsymbol{\tau},
+\end{equation*}
+where $\boldsymbol{t}$ denote the total traction, $t_N$ is the normal traction, $t_{T}$ represents the tangential traction, $\boldsymbol{n}$ and $\boldsymbol{\tau}$ are the normal and tangential unit vectors, respectively. The normal contact constraint is the same for these three contact types and are detailed in [contact module description](/modules/contact/index.html). The differences among these contact types lie in the constraint equations for the tangential contact, which is described in the following section.
+
+### Contact Models
+
+The `glued` contact does not allow slip between the nodes once they come into contact, i.e., $|\boldsymbol{v}_{T}| = 0$,
+where $\boldsymbol{v}_{T}$ is the tangential relative velocity.
+ Due to the no-slip condition, the glued contact is often used to model bonded connections in e.g.,  structural analysis.
+
+In contrast to the `glued` contact, the `frictionless` contact allows slip between the nodes that are in contact. However, no tangential traction is assumed, i.e., $|\boldsymbol{v}_{T}| > 0$ and ${t}_{T} = {0}$. The frictionless contact is often utilized when the contact surface is pretty smooth thus tangential fraction along the interface can be omitted.
+
+Different from the above two contact models, the `coulomb` model accounts for the friction along the interface caused by the relative motion between nodes. This is modeled using the Coulomb law, which computes a tangential traction that is proportional to the normal traction via a slip condition:
+\begin{equation*}
+{t}_{T} = -\mu |t_N| \frac{\boldsymbol{v}_T}{||\boldsymbol{v}_T||}\cdot\tau \quad \text{if} \quad ||{t}_T||> \mu |t_N|,
+\end{equation*}
+where $\mu$ is the friction coefficient.
+
+### Contact Formulations
+
+To enforce the contact constraints, three types of formulations are available. These are `kinematic`, `penalty`, and `mortar`.
+The `kinematic` and `penalty` formulations have been developed for some time and turn out to be robust for many mechanical contact problems. Recent efforts have been put in developing a mortar-based contact formulation. In this documentation, however,  we focus on `kinematic` and `penalty` formulations. For readers who are interested in mortar-based mechanical contact, please go to [mortar constraint system](Constraints/index.md) for more information.
+
+The overall workflow of imposing a mechanical constraint is as follows: 1) Determine whether penetration occurs. If so, constrain displacement; 2) In the normal direction, eliminate the penetration; 3) In the tangential direction, slip must satisfy friction model, no slit is allowed for the `glued` model, no constraint is applied for the `frictionless` model, Kuhn-Tucker conditions must be satisfied for the `coulomb` model. The mechanical contact enforcement steps for the `glued` and `frictionless` models are summarized below:
```

Comment:
slit -> slip

Reply:
Kuhn-Tucker conditions must be satisfied in the tangential direction for the `coulomb` model.